assets/modules.json

```diff
@@ -837,12 +879,18 @@
         "name": "FindSDL_ttf",
         "description": "Locate SDL_ttf library"
     },
-    "FindSelfPackers": { "name": "FindSelfPackers", "description": "Find upx" },
+    "FindSelfPackers": {
+        "name": "FindSelfPackers",
+        "description": "Find upx"
+    },
     "FindSQLite3": {
         "name": "FindSQLite3",
         "description": "Find the SQLite libraries, v3"
     },
-    "FindSquish": { "name": "FindSquish", "description": "-- Typical Use" },
+    "FindSquish": {
+        "name": "FindSquish",
+        "description": "-- Typical Use"
```

Comment:
What does this mean?

Reply:
It's parsed from the cmake docs, it's likely that we could use some improvement there, but for now it's the first paragraph from the cmake docs.